### ISSUE TITLE
5Е organization improvements

### ISSUE THREAD

**Issue by apoapostolov** [espositos/fvtt-tokenactionhud#304]
Few simple breakdowns of the Item type may be needed for 5E to make it clearer, especially when used during combat. Although there's now support for Action, Bonus Action, and Special Actions, for those who do not use that breakdown the current order is a bit cumbersome especially in the mid-tier where players have a lot of inventory items to use. Worse, static magic items get priority over the Inventory group where the weapons are (even the Magical weapons!) the most used part of the Inventory. Also, the Inventory tab is often overloaded with items in later games making it unpleasant to look at during every turn where the player's only intent is to use a weapon. 

* Create a new group called Weapons, exclude them from Items, and put it first in order. This will make it just as simple and logical as Strikes in PF2 and avoid piling weapons with all other usable items. It makes sense to include all Ammunition type items here as well.
* Move Magical Items after Weapons but before Inventory. Usually, magical weapons count as Weapons and do not appear in Magical Items. They will appear in the Weapons group. 
* I would also suggest, but it is nice to have and not as critical, to put Consumables as a new group that is separate from Inventory and that is between Magic Items and Inventory, to make it much more helpful to players to find healing potions, etc.

To sum up, the order for 5E should be:
* Weapons - only the weapon group from Items with Weapon type and Consumable type Ammunition sub-type
* Magical Items - all magical items to either remind them during combat or with activations
* Inventory - all inventory with Armor, Tools, Consumables, etc. 
* ...all other tabs, in current order...
